### PR TITLE
SW-2124: Responsive Page Layouts -> Empty -> Seed Banks

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR includes changes to align several pages with the [empty state designs in figma](https://www.figma.com/file/aLIG1C6uIMyBWiTopSluxe/Visual-Refresh?node-id=853%3A112604&t=FSN08oFo2ACctbR6-0):

- Species
- Planting Sites
- Inventory
- Seed Banks
- Nurseries

## Screenshots

Desktop:

localhost_3000_ (1)

localhost_3000_ (2)

localhost_3000_ (3)

localhost_3000_ (4)

localhost_3000_ (5)

localhost_3000_

Tablet:

localhost_3000_(iPad Mini) (1)

Phone:

localhost_3000_(iPhone SE) (1)

### CODE REVIEW

File: src/components/common/TfMain.tsx [hunk 1]
```diff
@@ -2,23 +2,54 @@ import React from 'react';
...
+    [theme.breakpoints.up('xl')]: {
+      background: (props: StyleProps) =>
+        !props.backgroundImageVisible
+          ? 'initial'
+          : 'url(/assets/home-bg-right-layer-z4.svg) no-repeat 100% 100%/auto 285px, ' +
+            'url(/assets/home-bg-left-layer-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-water-z2.svg) repeat-x 0 100%/auto 180px, ' +
+            'url(/assets/home-bg-left-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-right-z3.svg) no-repeat 100% 100%/auto 400px',
+      backgroundAttachment: 'fixed !important',
```
Comment (joshbuchea) [inline src/components/common/TfMain.tsx:40]: I'm not proud of the `!important` here, but there's a style rule that's overriding `background-attachment` higher up in the cascade somewhere.

Comment (karthikbtf) [reply src/components/common/TfMain.tsx:40]: i couldn't find the overriding class either.. maybe some default behavior.

**Comment by karthikbtf** [comment]
Looks good Josh, I see these screens for inventory and accessions empty states.. am I missing something?

Terraware App 2022-11-23 14-28-45

Terraware App 2022-11-23 14-28-28

**Comment by hannah-cai** [comment]
looks good! just one note: in the third screenshot, there shouldn't be a header ("Seed Bank Setup") under the image if there's no description.

**Comment by joshbuchea** [comment]
@karthikbtf thanks for highlighting these unfinished pages.

The accession page empty state wasn't included in this PR, only the pages mentioned in the description. And the inventory page empty state looks different than the one I was checking while working on this locally 🤔

The accession page empty state, and the inventory page empty state if needed, will be covered in a follow-up PR.

**Comment by joshbuchea** [comment]
@hannah-cai ok got it. It's no problem to include a description if we have/want one for the page, do you know who I would go to for copy

**Comment by hannah-cai** [comment]
> @hannah-cai ok got it. It's no problem to include a description if we have/want one for the page, do you know who I would go to for copy?

ah I meant to just take out the header, I don't think we need a header or description when there's only one image

File: src/components/common/TfMain.tsx (hunk 1 above)
Comment (joshbuchea) [reply src/components/common/TfMain.tsx:40]: Thanks for looking @karthikbtf 🙌

**Comment by joshbuchea** [comment]
@hannah-cai ok got it, thanks for clarifying. This can be resolved in the content by choosing one of the following two options:

- Add a description to the "Seed Bank Setup" item OR
- Remove the "Seed Bank Setup" title

Performing either of the above actions will resolve the issue of showing a title without a related description.

I'll check with Allison to see which option she prefers.

**Review by karthikbtf — APPROVED** [review]
looks so much nicer than `/main`

**Pull request merged**